src/amber/controller/helpers/route.cr

```diff
@@ -0,0 +1,23 @@
+module Amber::Controller::Helpers
+  module Route
+    def route_action
+      context.request_handler.action
+    end
+
+    def route_resource
+      context.request_handler.resource
+    end
+
+    def route_scope
+      context.request_handler.scope
+    end
+
+    def route_controller
+      context.request_handler.controller
+    end
+
+    def controller_name
+      self.class.name.gsub(/Controller/i, "")
```

Comment:
For this regex, I would anchor to the end. And does the `i` make it case-insensitive, is that necessary?

https://play.crystal-lang.org/#/r/31bu

Reply:
Would people want the controller name downcased?

Reply:
Yeah, I think we should use `/Controller$/`

Reply:
You guys are correct I will also fix the regex. I looked at the rails implementation 

https://apidock.com/rails/ActionController/Metal/controller_name/class

Reply:
Use sub instead of gsub since we only want to replace once.